tee-worker/cli/src/trusted_base_cli/commands/set_balance.rs

```diff
@@ -48,7 +51,7 @@ impl SetBalanceCommand {
 
 		let (mrenclave, shard) = get_identifiers(trusted_args, cli);
 		let nonce = get_layer_two_nonce!(signer, cli, trusted_args);
-		let top: TrustedOperation = TrustedCall::balance_set_balance(
+		let top: TrustedOperation<TrustedCallSigned, Getter> = TrustedCall::balance_set_balance(
```

Comment:
Noticed some part of the cli code using `let xxx = TrustedOperation<x,x>(Trustedxxx::xx())`. But here using `let xxx: TrustedOperation<x,x> = Trustedxxx:xx()`. Should keep the consistency? 
e.g. [tee-worker/cli/src/evm/commands/evm_read.rs](https://github.com/litentry/litentry-parachain/pull/2281/files#diff-a0657c51fc79b2f626549e863cb50069d30886c4a23e565b1c4a00f581bc7885)

Reply:
Good catch - I'll change it